frontend/src/types/ontologyTypes.ts

```diff
@@ -11,6 +11,12 @@ export type SustainabilityGoal = {
   icon: string;
 };
 
+export type SubGoal = {
```

Comment:
Similar to the comment on the backend type, this type should be derived from Node and add just a description field.

Reply:
And that description must be nullable.

Reply:
Are you sure it should be derived from Node? The subgoal query might/should get expanded with more data such as its correlating icons

Reply:
I think so. I mean, do it like this 

`export interface Subgoal extends Node {
description? string;
}`

and other possible additional fields.